traffic_ops/testing/api/v3/tc-fixtures.json

```diff
@@ -2317,7 +2317,7 @@
             "xmppPasswd": "X"
         },
         {
-            "cachegroup": "parentCachegroup",
+            "cachegroup": "parentCachegroup2",
```

Comment:
Why does the test fail when `atlanta-mid-17` is in `parentCachegroup`?

Reply:
parentCachegroup was used as part of a topology so i just put it in another cg so not to mess w/ that test. but all passes now.

Reply:
It shouldn't matter that it is used as part of a topology. `atlanta-mid-17` is in CDN `cdn2`. The delivery service used in the test has xmlId `ds-top` and is in CDN `cdn1`. So, `atlanta-mid-17` being returned in the response is a bug because it's in a different CDN, right?

Reply:
reverting that.... parentCachegroup is fine for my test...now to figure out why it's breaking the topology test..